tests/test_infra.py

```diff
@@ -132,3 +138,12 @@ def test_bashcompletion_all_commands():
         real_command_names.update(cmd.name for cmd in cmds)
 
     assert completed_commands == real_command_names
+
+
+def test_black():
+    """Verify all files are nicely styled."""
+    proc = subprocess.run(
+        ["black", "--check", "--diff", "."], check=False, capture_output=True, text=True
```

Comment:
Can we use black programatically (and not as a subscript)? this is breaking my development setup (I don't run the tests from inside the VM, so "black" is not in the path)

Reply:
I can give it a shot.

Reply:
Done!